.github/workflows/smoketest.yml

```diff
@@ -0,0 +1,23 @@
+name: Smoketest
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+defaults:
```

Comment:
Is this `defaults` section needed? I don't see it on our other actions yamls.

Reply:
Good eye. I'd used the [brim.yml](https://github.com/brimsec/zeek/blob/master/.github/workflows/brim.yml) from our Zeek fork as my template, and it happened to include that section. I'm assuming that had been there because the Zeek build action runs on both UNIX-style and Windows hosts, and the Windows ones have the option to run on alternates like PowerShell. You're right that it's surely needed, so I've just pushed a commit to remove it, and once everything runs clean I'll merge the PR.